### PR TITLE
Admin panel

### PR DESCRIPTION
**Pull request by Maciorex**
Creating functionality to admin panel, mostly connected with CRUD for disciplines.

### CODE REVIEW

**Review by ozimeu — APPROVED** [review]
good job! 👌

File: app/controllers/disciplines_controller.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class DisciplinesController < ApplicationController
+  before_action :require_admin, only:[:new, :create, :edit, :update, :destroy]
```
Comment (ozimeu) [inline app/controllers/disciplines_controller.rb]: space after `only:`

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class UsersController < ApplicationController
+
```
Comment (ozimeu) [inline app/controllers/users_controller.rb]: empty line

File: app/views/disciplines/show.erb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<h1 align="center"><%= @discipline.name %></h1>
+
+<div class="well">
+  <div align="center">
+    <%= @discipline.description %>
+  </div>
+</div>
+<% if current_user && current_user.admin? %>
+<%= link_to "edit discipline", edit_discipline_path(@discipline), class: "btn btn-xs btn-primary" %>
```
Comment (ozimeu) [inline app/views/disciplines/show.erb]: the link line should be indented as it's a child of the if statement

File: app/controllers/disciplines_controller.rb (hunk 1 above)
Comment (Maciorex) [reply app/controllers/disciplines_controller.rb]: yaaaaz

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (Maciorex) [reply app/controllers/users_controller.rb]: kill it!

File: app/views/disciplines/show.erb (hunk 3 above)
Comment (Maciorex) [reply app/views/disciplines/show.erb]: si correcto

**Review by ozimeu — APPROVED** [review]
:+1:

**Pull request merged**